db/db_test.go

```diff
@@ -328,9 +328,6 @@ var _ = Describe("Database operation test", func() {
 				})
 
 				It("Updates the resource properly", func() {
-					By("Not allowing to update some fields")
-					Expect(networkResource1.Update(map[string]interface{}{"id": "new_id"})).ToNot(Succeed())
```

Comment:
Why is this removed?

Reply:
this logic has been moved from `Update` to `UpdateResourceInTransaction` which calls `Update`